master/internal/rm/kubernetesrm/pods.go

```diff
@@ -789,7 +811,11 @@ func (p *pods) startNodeInformer() error {
 }
 
 func (p *pods) startEventListeners() error {
-	for namespace := range p.namespaceToPoolName {
+	ns, err := workspace.GetAllNamespacesForRM(context.Background(), p.clusterName, p.namespace)
+	if err != nil {
```

Comment:
can we call this upfront once and pass the slice of namespaces to each informer?

Reply:
done